src/main/java/com/softserve/algorithms/Algorithms.java

```diff
@@ -54,4 +55,28 @@ else if (i < m)
             return count[n];
         }
     }
+
+    /**
+     *
+     * @param posts - number of posts
+     * @param colors - number of colors
+     * @return count of ways to color n posts using k colors
+     */
+    public static BigDecimal countWaysPaintingTheFence(int posts, int colors) {
```

Comment:
https://github.com/Java364/AlgorithmsCLI/blob/c352bc875dc33eb9d3bb84f393e62627a3502e0b/src/main/java/com/softserve/algorithms/Algorithms.java#L63-L65
Add `throws` clause to your method and add `@throws` tag to method's javadoc.

Reply:
Added `throws` and changed javadoc